superset-frontend/src/dashboard/components/SliceAdder.jsx

```diff
@@ -302,6 +357,25 @@ class SliceAdder extends React.Component {
             placeholder={t('Sort by')}
           />
         </Controls>
+        <div
+          css={theme => css`
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            padding: 0 ${theme.gridUnit * 3}px ${theme.gridUnit * 3}px
+              ${theme.gridUnit * 3}px;
+
+            & > span {
+              margin-right: ${theme.gridUnit}px;
+            }
+          `}
+        >
+          <Checkbox
+            onChange={this.onAccessibleCharts}
+            checked={this.state.showAccessibleCharts}
+          />
+          {t('Show other charts I have access to')}
```

Comment:
not sure how others feel, but to me "other" sounds like the complement rather than the union which I'd want the checkbox to expose:
```suggestion
          {t('Show all charts I have access to')}
```

Reply:
You're right. "other" does look like a complement and it should be the union.